### PR TITLE
Fixed margin issue with Sync note on different browsers

### PR DESCRIPTION
**Pull request by cedricium**
## Description:
Just downloaded Firefox Dev edition and started testing Notes on there. When
opening up the Sync note, I saw that there was a great deal of margin to it's
right (the note was not the full width of the sidebar like the Sync button).

On the regular edition of Firefox, the Sync note **was** the full width of the
sidebar. These small changes make it so that the Sync note is the full width of
the sidebar on both browsers.

## Images

**FF Dev edition - before:**
image

**FF regular edition - before:**
image

**FF Dev edition - after:**
image

**FF regular edition - after:**
image

### CODE REVIEW

File: src/sidebar/styles.css [hunk 1]
```diff
@@ -83,9 +83,9 @@ footer {
 
 #sync-note {
   display: none;
-  padding: 8px 4% 15px 4%;
+  padding: 8px 0% 15px 0%;
```
Comment (vladikoff) [inline src/sidebar/styles.css]: @cedricium you can drop `%` in the `0` values

Comment (cedricium) [inline src/sidebar/styles.css]: Good catch!

**Comment by vladikoff** [comment]
thank you @cedricium !!

**Comment by cedricium** [comment]
No, thank you @vladikoff! Everyone working on Notes has been really kind, helpful, and patient with me and its been an awesome experience helping out. 😄

**Comment by vladikoff** [comment]
@cedricium awwww yesss! We want to make sure to include you as a contributor in the `Brought to you by` section once this experiment goes live.

Example: https://testpilot.stage.mozaws.net/experiments/snooze-tabs

Please let me know if we can use your name and let me know which photo / avatar we should use on that page.

**Comment by cedricium** [comment]
@vladikoff that's such an honor, by all means go ahead and include my name (Cedric Amaya). And you can use my GitHub avatar. Thanks!

**Pull request merged**